src/test/java/stringcalculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,77 @@
+package stringcalculator;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class StringCalculatorTest {
+
+	@ParameterizedTest
+	@DisplayName("빈 문자열 또는 null을 입력할 경우 0을 반환해야 한다.")
+	@ValueSource(strings = {" ", "  "})
+	@NullAndEmptySource
+	void constructorWithNullOrBlank(final String input) {
+		// given
+		StringCalculator calculator = new StringCalculator(input);
+
+		// when / then
+		assertThat(calculator.calculate()).isEqualTo(NonNegativeInteger.ZERO);
```

Comment:
테스트 코드를 given when then으로 나눠서 작성해보라고 말씀드렸던건 아래처럼 (when 절과 then 절의 분리해서) 작성하는 것을 말씀드리는거였어요! 

```suggestion
		// given
		StringCalculator calculator = new StringCalculator(input);

                // when
                NonNegativeInteger actual = calculator.calculate()

		// then
		assertThat(actual).isEqualTo(NonNegativeInteger.ZERO);
```

Reply:
아 아예 코드로 단계를 나눠보라는 말씀이셨군요. 반영하겠습니다.